web-console/4.0-designs/containers/containers.md

```diff
@@ -0,0 +1,79 @@
+# Containers
+
+## Resource Details: Containers Section
+
+- The following resource types will include a "Containers" section on their object detail pages.
+  - Pods
+  - Deployments
+  - Deployment Config
+  - Replica Sets
+  - Stateful Sets
+  - Cron Jobs
+  - Jobs
+  - Daemon Sets
+  - Replication Controllers
+- The Containers section will always be displayed below the general overview details.
+- The Containers section consists of a table with 4-6 columns, depending on the resource type.
+
+**Pod Details Page**
+![pods](img/pod.png)
+- For Pods, the columns include the following:
+  - Name
+  - Image
+  - State
+  - Restarts
+  - Started
+  - Finished
+- The Name column should use the Container badge and be shown as a link to the container details page.
+
+**Deployment Details Page**
+![deployment](img/deployment.png)
+
+- For all other resources, the columns include the following:
+  - Name
+  - Image
+  - Resource Limits
+  - Ports
+- The Name column should use the Container badge and be shown as a link to the container details page.
+
+## Container Details
+
+- The container details page is accessed via one of the resource details pages mentioned above (pods, deployments, stateful sets, etc.)
+- The page includes the following sections:
+  - Overview
+  - Image Details
+  - Mounted Volumes
+  - Environment Variables
+
+### Overview Section
+
+**Container Details with Pod Owner**
+![pod-container](img/pod-container.png)
```

Comment:
@beanh66 should we have a breadcrumb here ?

Reply:
@serenamarie125 In the future, definitely. I just did not put breadcrumbs yet since I didn't think we had them for 4.0.